clients/fides-js/src/components/EthycaLogo.tsx

```diff
@@ -0,0 +1,9 @@
+import { h } from "preact";
+
+const EthycaLogo = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="31" height="20" fill="#2b2e35">
```

Comment:
let's set this fill color with a CSS class instead, so you can just change a var like `--fides-overlay-brand-link-font-color` to style this

an inline style fill color like this is hard to change